src/bundle/Resources/views/themes/admin/ui/layout.html.twig

```diff
@@ -221,6 +221,8 @@
             {% endblock %}
         {% endif %}
 
+        {{ ibexa_render_component_group('html-body') }}
```

Comment:
Sidenote: `html-body` component group should be added to https://doc.ibexa.co/en/latest/administration/back_office/back_office_elements/custom_components/